.eslintignore

```diff
@@ -1 +1,2 @@
 /.eslintrc-internal.js
+/copyright.js
```

Comment:
We have to ignore the template, otherwise ESLint will complain that it doesn't have the right headers (ie. because it contains the string `YEAR` instead of the expected `2020` — or `2021`, `2022` etc).